Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -223,6 +232,20 @@ def send_report(self):
     def setup_layout(self):
         self.add_learner_name_widget()
         self.add_main_layout()
+        # Options specific to target variable type, if supported
+        if self.__handles_any_input_type():
+            # Only add a classification section if the method is overridden
+            if type(self).add_classification_layout is not \
+                    OWBaseLearner.add_classification_layout:
```

Comment:
@astaric, we're seeing more and more of this checking whether a class has overrides an inherited method. (Not that it's nice, but it's useful.) Is it time to implement something like
```
def overrides(method, base):
    return getattr(type(method.__self__), method.__name__) != getattr(base, method.__name__)
```
to be used like (here above) `if overrides(self.add_classification_layout, OWBaseLearner)`?